perun-core/src/main/java/cz/metacentrum/perun/core/bl/UsersManagerBl.java

```diff
@@ -1216,6 +1216,17 @@ void deletePassword(PerunSession sess, User user, String loginNamespace)
 	 */
 	void checkReservedLogins(PerunSession sess, String namespace, String login, boolean ignoreCase) throws AlreadyReservedLoginException;
 
+	/**
+	 * Check if login is blocked. Login can be blocked by default (used by internal components),
+	 * globally or in namespace.
+	 *
+	 * @param sess session
+	 * @param userLogin login
+	 * @param attribute attribute
+	 * @throws LoginIsAlreadyBlockedException when login is blocked
+	 */
+	void checkBlockedLogins(PerunSessionImpl sess, String userLogin, Attribute attribute) throws LoginIsAlreadyBlockedException;
```

Comment:
one more thing - it would make sense to me pass login and namespace parameters directly instead of login and attribute containing the namespace, so it is also usable when we'd just like to check login without namespace etc.

also - i see reserved logins can ignore case of the login - maybe we'd have a use case for that also here? should discuss it...